extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -132,6 +130,16 @@ function () {
 					];
 				} );
 
+				foreach ( $validAdminIds as $adminId ) {
+					if( !in_array( $adminId, array_column( $result,  'userId' ) ) ){
```

Comment:
I'm trying to do this but I can't modify the existing array (existing in getAllAdmins) inside sqlData->runloop because it has different scope and as it build the it's own data from scratch.